narwhal/primary/src/core.rs

```diff
@@ -479,6 +480,16 @@ impl Core {
             .header_to_certificate_latency
             .with_label_values(&[&epoch.to_string()])
             .observe(created_at.elapsed().as_secs_f64());
+
+        #[cfg(feature = "benchmark")]
+        // NOTE: This log entry is used to compute performance.
+        tracing::info!(
+            "Header {:?} took {} seconds to be materialized to a certificate {:?}",
+            certificate.header.digest(),
+            created_at.elapsed().as_secs_f64(),
```

Comment:
I think this calculation should be `certificate.metadata.created_at - certificate.header.created_at`, because commit time is calculated from `certificate.metadata.created_at`?